### PR TITLE
Update ytdl-sub (headless) template

### PR DESCRIPTION
**Pull request by jmbannon**
Update various fields + paths for the ytdl-sub (headless) variant. Not sure if adding/removing paths changes anything - input on this would be great

### CODE REVIEW

File: templates/ytdl-sub.xml [hunk 1]
```diff
@@ -5,20 +5,17 @@
...
-  <Config Name="tmp" Target="/tmp" Default="" Mode="rw" Description="It is a temp folder, obviously this is for permanent files." Type="Path" Display="always" Required="false" Mask="false">/mnt/user/media/temp/</Config>
-  <Config Name="Media" Target="/media" Default="" Mode="rw" Description="Where you store your media, duh" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/media</Config>
-  <Config Name="config" Target="/config" Default="" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata</Config>
+  <Config Name="config" Target="/config" Default="" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/ytdl-sub</Config>
+  <Config Name="TV Shows" Target="/tv_shows" Default="" Mode="rw" Description="Path to store TV shows" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/tv_shows</Config>
+  <Config Name="Music" Target="/music" Default="" Mode="rw" Description="Path to store music" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/music</Config>
```
Comment (nwithan8) [inline templates/ytdl-sub.xml:16]: Splitting the single `/media` folder into three `/music`, `/tv_shows` and `/music_videos` folders is going to be a breaking change for any user updating their Docker container with this template.

I would assume anyone using this template already has their folders subdivided inside the container itself as a result. Could we consider maintaining the single `/media` folder?

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/ytdl-sub.xml (hunk 1 above)
Comment (jmbannon) [reply templates/ytdl-sub.xml:18]: Done - kept the music/tv_shows/etc since they're optional. That should be safe, right?

Comment (jmbannon) [reply templates/ytdl-sub.xml:16]: friendly bump @nwithan8

**Review by nwithan8 — APPROVED** [review]

File: templates/ytdl-sub.xml [hunk 2]
```diff
@@ -5,20 +5,19 @@
...
-  <Support>https://forums.unraid.net/topic/138060-support-catduck-ytdl-sub-youtube-downloading-with-meta-data-for-plexkodijellyfin/</Support>
+  <Support>https://discord.gg/235bWeyffD</Support>
   <Project>https://github.com/jmbannon/ytdl-sub</Project>
-  <Overview>Automate downloads and metadata generation with YoutubeDL&#xD;
-&#xD;
-&#xD;
-Please take your time to go through the walkthrough&#xD;
-&#xD;
-https://github.com/jmbannon/ytdl-sub/wiki</Overview>
+  <Overview>Automate downloads and metadata generation with YoutubeDL. This is the headless variant. See `ytdl-sub-gui` for a web-gui version.</Overview>
```
[the remark this answers is not in the document]
Comment (jmbannon) [reply templates/ytdl-sub.xml:10]: The GitHub homepage is the better landing now - the wiki and readthedocs are now moreso for advanced users who want to make custom configs

File: templates/ytdl-sub.xml [hunk 3]
```diff
@@ -5,20 +5,19 @@
...
+  <Config Name="TV Shows" Target="/tv_shows" Default="" Mode="rw" Description="Path to store TV shows" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/tv_shows</Config>
+  <Config Name="Music" Target="/music" Default="" Mode="rw" Description="Path to store music" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/music</Config>
+  <Config Name="Music Videos" Target="/music_videos" Default="" Mode="rw" Description="Path to store music videos" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/music_videos</Config>
+  <Config Name="Movies" Target="/movies" Default="" Mode="rw" Description="Path to store movies" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/movies</Config>
+  <Config Name="Media" Target="/media" Default="" Mode="rw" Description="Top-level folder for all media, can use separate mounts instead" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/media</Config>
+  <Config Name="tmp" Target="/tmp" Default="" Mode="rw" Description="Temp folder for ytdl-sub to use as a cache" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/media/temp/</Config>
```
Comment (jmbannon) [inline templates/ytdl-sub.xml:20]: @nwithan8 @Roxedus Would it be safe to remove this since it's optional?

Comment (nwithan8) [reply templates/ytdl-sub.xml:20]: I would imagine so. I don't think an end-user would ever need to access the container's cache folder, so it doesn't need to be mapped to a host folder.

Comment (jmbannon) [reply templates/ytdl-sub.xml:20]: Agreed - it only causes permission issues for our users. Will make a PR to remove it

**Pull request merged**